src/components/ActorsTodo/ActorsTodo.tsx

```diff
@@ -0,0 +1,69 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import * as S from "./ActorsTodo.Styled";
+import { useState } from "react";
+
+interface TypesForActors {
+	id: number;
+	name: string;
+}
```

Comment:
Lepszą nazwą byłoby tutaj `Actor` albo `ActorTypes` bo definiujemy tutaj pojedyńczego aktora.
Gdybyśmy tutaj typowali tablicę aktorów to moglibysmy dać `ActorsTypes` wtedy nazwa skazuje na typ dla arrajki.